src/test/app/Regression_test.cpp

```diff
@@ -247,6 +250,100 @@ struct Regression_test : public beast::unit_test::suite
             jrReader.parse(jvRequest, buffers) && jvRequest.isObject());
     }
 
+    void
+    testInvalidTxObjectIDType()
+    {
+        testcase("Invalid Transaction Object ID Type");
+        // Crasher bug introduced in 2.0.1. Fixed in 2.3.0.
+
+        using namespace jtx;
+        Env env(*this);
+
+        Account alice("alice");
+        Account bob("bob");
+        env.fund(XRP(10'000), alice, bob);
+        env.close();
+
+        auto index = [&](auto account) {
+            Json::Value req(Json::objectValue);
+            req[jss::account] = account;
+            auto const resp = env.rpc("json", "account_info", to_string(req));
+            BEAST_EXPECT(
+                resp.isMember(jss::result) && resp[jss::result].isObject());
+            auto const& result = resp[jss::result];
+            BEAST_EXPECT(
+                result.isMember(jss::account_data) &&
+                result[jss::account_data].isObject());
+            auto const& ad = result[jss::account_data];
+            BEAST_EXPECT(ad.isMember(jss::index) && ad[jss::index].isString());
+            uint256 index;
+            BEAST_EXPECT(index.parseHex(ad[jss::index].asString()));
+            return index;
+        };
```

Comment:
I think this can be greatly simplified by using `keylet::account(alice)`

Reply:
excellent idea. That will make this whole lambda unnecessary. I added the necessary proposed changes.

Reply:
> I think this can be greatly simplified by using `keylet::account(alice)`

Thanks @mvadari. That was a great observation and suggestion.